omnigibson/envs/env_base.py

```diff
@@ -417,20 +440,19 @@ def update_task(self, task_config):
         og.sim.stop()
         self._load_task(task_config=task_config)
         og.sim.play()
-        self.reset()
 
         # Load obs / action spaces
         self.load_observation_space()
         self._load_action_space()
 
+        self.reset()
+
         # Scene is now loaded again
         self._loaded = True
 
     def close(self):
-        """
-        Clean up the environment and shut down the simulation.
-        """
-        og.shutdown()
+        """No-op to satisfy certain RL frameworks."""
+        pass
```

Comment:
hmm interesting, do we not want to shutdown?

Reply:
I suppose since we could have multiple environments, calling `close` on one environment should not shutdown everything.

Reply:
I assume this is because `og.shutdown()` automatically terminates the entire python process

Reply:
Yup. No code anticipates that the process will get nuked when env.close() is called, so that's not an option. We can invalidate the simulation state later somehow (e.g. clear the env) but there's not really a reason for this to do anything.